src/crypto/testdata/rsa_2048_private_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Are there safety checks around submitting private keys into the repo?

Based on the path, this key is for testing, but in the past, keys have been exposed through error.

Reply:
We do not. =/

If you have presubmit suggestions I'm happy to hear them. Can you file an issue?

Reply:
https://github.com/lowRISC/opentitan/issues/5843